stories/assets/styles/global/global.scss

```diff
@@ -295,3 +317,79 @@ address {
   line-height: 1.2;
   margin: 0;
 }
+
+// ==========================================================================
+// Grid
+// @TODO: Separate into it's own component if we need a full 12-column grid.
+// ==========================================================================
+
+// The project's active USWDS breakpoints;
+$project-active-uswds-breakpoints: ();
+
+// If it's enabled, we'll add it to the list.
+@each $breakpoint, $size in $theme-utility-breakpoints-complete {
```

Comment:
@mejiaj I'm getting compilation errors when I try to build this in Drupal with `lando gulp`

![image](https://github.com/user-attachments/assets/c3e0b9e3-6e32-422a-af8d-a73ada403c5a)

It seems to work fine in storybook, but in Drupal compilation fails. I'm not sure how they are different or how to properly import `node_modules/@uswds/uswds/packages/uswds-core/src/styles/settings/_settings-utilities.scss`

I get the same error in platform.sh when it tries to build:
![image](https://github.com/user-attachments/assets/aa361983-fc6f-4f3f-9f31-ade334aaa96a)